xpk.py

```diff
@@ -158,6 +359,26 @@
       resources:
       - name: "{resource_type}"
         nominalQuota: {total_chips}  # Number of slices * number of chips in each slice
+  - coveredResources: ["cpu", "memory"]    
+    flavors:
+    - name: cpu-rm
+      resources:
+      - name: "cpu"
+        nominalQuota: 6
+      - name: "memory"
+        nominalQuota: 10G
```

Comment:
What is the diff between the resources specified here and in the yaml above?

Reply:
ResourceFlavors are mostly the nodeLabels, in case of CPUs, they are just the nodepool names. 
Here we specify what kind of resources the cluster queue should expect, which in case of Pathways is TPUs + CPUs and memory.

These are the resources I learned from -
1. https://cloud.google.com/kubernetes-engine/docs/tutorials/kueue-intro#create_the_resourceflavor - more on resource flavor
2. https://kueue.sigs.k8s.io/docs/tasks/administer_cluster_quotas/#1-create-a-clusterqueuedocsconceptscluster_queue  - resources in cluster queue.

LMK if you see an issue!